### DISCUSSION TITLE
Support for Emoji in the Toolbar while writting a comment or starting new Discussion.

### DISCUSSION THREAD

**Discussion by BoQsc** [community/community#3026] [Discussions]
I'd like to see a button where I could click and select from many Emojis to insert.
![image](https://user-images.githubusercontent.com/21064622/102630375-2729ab80-4155-11eb-9c8c-dc5bb4b8ea26.png)

**Comment by evi-liu** [comment]
Did you know you can select any emoji using the following keyboard shortcuts? 

**Windows** - windows key + .
**Mac** - control + command +space

**Reply by patepelo** [reply]
Yes, but GitHub has its own rendering/interpretation of the emojis. Would be useful anyways. Thanks for the tip anyways :)

**Reply by patepelo** [reply]
Another alternative is to type colon and start typing search tearms and emojis appear. 😸

**Comment by PranamBhat** [comment]
Hello!

You can use:

Windows logo key   + . (period). The emoji keyboard will appear.

Check out more here : [https://support.microsoft.com/en-us/windows/windows-10-keyboard-tips-and-tricks-588e0b72-0fff-6d3f-aeee-6e5116097942](https://support.microsoft.com/en-us/windows/windows-10-keyboard-tips-and-tricks-588e0b72-0fff-6d3f-aeee-6e5116097942)
